### PR TITLE
Update timeout and delay time for lease during upload

### PR DESCRIPTION
**Pull request by karajas**
/cc @weshaggard @wtgodbe

### CODE REVIEW

File: src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs [hunk 1]
```diff
@@ -133,7 +133,7 @@ public BlobFeedAction(string expectedFeedUrl, string accountKey, string indexDir
 
             await clientThrottle.WaitAsync();
             string leaseId = string.Empty;
-            AzureBlobLease blobLease = new AzureBlobLease(feed.AccountName, feed.AccountKey, string.Empty, feed.ContainerName, uploadPath, Log, "60", "10");
+            AzureBlobLease blobLease = new AzureBlobLease(feed.AccountName, feed.AccountKey, string.Empty, feed.ContainerName, uploadPath, Log, "15", "3000");
```
Comment (weshaggard) [inline src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs]: Can you add a comment in the commit to state what these new times are? Are they all in milliseconds? which one is delay and which one is timeout.

Comment (karajas) [reply src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs]: Sure, the first one is in seconds and the second in milliseconds.

**Comment by karajas** [comment]
The only way to test that this works is to ensure that it works in the context of official build, testing there and will update with results.

**Comment by karajas** [comment]
This is now working in the intended manner in the CoreCLR build.

**Pull request merged**